docs: [DHIS2-12337] update saving tei docs (#3203)
dhis2/capture-app: docs/user/using-the-capture-app.md

    the field. This will open a map where you can search for a location and capture
    a polygon (button in the upper right corner of the map).

7. Click **Save new** to register the tracked entity instance. 

>     The **Cancel** button will be disabled, and a spinner on the **Save** button will indicate that the request has been sent.

Reviewer: ```suggestion
    The **Save** button shows a loading spinner and the **Cancel** button is disabled while the request is processing.
```

Reviewer 2: Thanks for the review @cooper-joe. The docs are now updated with your suggestions.

5. You will now be able to see the warning produced by the program rule underneath the birth date field. 
![](resources/images/program-rules-on-creation-02.png)

### Leaving the page

If the user attempts to change their context, either by changing the program or the organisation unit from the locked selector or by clicking the **Clear selections** button the outcome will depend on the page state:
1. The user has not typed anything in the fields: The context is updated without any message and the user is redirected.
2. The user has already pressed the **Save** button, and a new registration has been sent to the server, the redirection to enrollment dashboard page will be cancelled and user will be navigated to the new context page. 
3. The user has entered details into the fields, see a modal warning them that they will lose their data.

> 

Reviewer: This doesn't seem relevant to the user documentation, especially referring to `the user`. This is a technical description of the different departure flows possible from this page, but I don't think they need to be documented in user documentation. The user is informed from the interface.

(Aside: user documentation should not refer to "the user". That term is meant for technical design documentation.)
